pkg/daemon/constants/constants.go

```diff
@@ -108,4 +108,7 @@ const (
 
 	// CRIOServiceName is used to specify reloads and restarts of the CRI-O service
 	CRIOServiceName = "crio"
+
+	// DaemonReloadServiceName is used to specify reloads and restarts of the systemd manager configuration
+	DaemonReloadServiceName = "daemon-reload"
```

Comment:
nit: I guess this isn't technically a service. Maybe something like DaemonReloadCommand ?